README.md

```diff
@@ -1,3 +1,48 @@
+## Math Magicians
+
+## Built With
```

Comment:
- [ ]  Please, add a meaningful  `Project description. Remember A project description is a high-level overview of why you're doing a project. The document explains a project's objectives and essential qualities. Is an essential document that provides your audience with a comprehensive overview of the project. Think of it as the elevator pitch that focuses on what and why without delving into how. 😉 💯